packages/fx-core/src/common/deps-checker/depsChecker.ts

```diff
@@ -56,6 +56,8 @@ export interface TestToolInstallOptions {
   symlinkDir: string;
   projectPath: string;
   versionRange: string;
+  // in milliseconds
```

Comment:
Could this be customized by user? If so, millisecond is a bit ... weird - I need to calculate how many milliseconds a week has, or 1000.000 how many zeros I need to add.
Could it be days and integer only?

Reply:
Added mainly for disable auto update. I will remove this property and let user use a fix version number `1.2.3` to skip auto update. This will align with func install

Reply:
sounds good.